Model/Api/UpdateCart.php

```diff
@@ -121,8 +125,56 @@ public function execute($cart, $add_items = null, $remove_items = null, $discoun
                 $this->setShipment($cart['shipments'][0], $immutableQuote);
                 $this->setShipment($cart['shipments'][0], $parentQuote);
             }
+            // TODO : add/remove giftcard 
```

Comment:
Please add TODO for clearing bolt order cache, we can implement it once finish the discussion about caching.

Reply:
Done